UVicMartletplace/martletplace
# Brent/backend385 fixing listing location

# Description
GET /api/listings and GET /api/listing were not retrieving the listings with the location in the correct format. The format resulting for the database query is ```"location": "(41.8781,-87.6298)"```, and was not formatted as specified in the API Agreement. This PR parses it into the correct format as shown below, and fixes the tests accordingly:

```
"location": {
  "latitude": 34.23551,
  "longitude": -104.54451
},
```

Closes #385 

## How to Test

1. Unit tests via ```npm test```
2. Run the application and see responses in devtools


## Checklist
